mitiq/zne/scaling/tests/test_folding.py

```diff
@@ -1415,7 +1032,7 @@ def test_default_weight():
 
 @pytest.mark.parametrize(
     "fold_method",
-    [fold_gates_from_left, fold_gates_from_right, fold_gates_at_random],
+    [fold_gates_at_random],
 )
```

Comment:
Can we remove the use of `pytest.mark.parametrize` here since we are only testing with one `fold_method` now? If you agree, in the tests below as well.

Reply:
I think this is a good idea to clean up some code. Do you think we should  change the names of the tests themselves to indicate that they all call `fold_at_random` or are we comfortable just assuming that as the default unless otherwise specified?

Reply:
I think if test names indicate that multiple fold methods are going to be invoked then the name should be changed to reflect this change. I did a quick pass through this file and nothing jumped out to me as needing renaming, but worth double checking.

Reply:
Yeah I agree. No need to rename.